Update docs (#235)
kyma-project/btp-manager: docs/certs.md

![Certification management diagram](./assets/certs.svg)

> The reconciliation of certifications can be triggered from the three events, first is normal scheduled reconciliation, second event is by doing edit on btpOperator CR, and the third is by use of custom watchers founded on Secrets and Webhooks resources.

Reviewer: ```suggestion
Certification reconciliation is triggered by one of the three events: scheduled reconciliation, editing btpOperator CR, or using custom watchers founded on Secret and Webhook resources.
```
